pkg/controller/windowsmachineconfig/windows/windows.go

```diff
@@ -112,38 +109,34 @@ func (vm *Windows) initializeBootstrapperFiles() error {
 	// Download the worker ignition to C:\Windows\Temp\ using the script that ignores the server cert
 	ignitionFileDownloadCmd := wgetIgnoreCertCmd + " -server " + vm.workerIgnitionEndpoint + " -output " +
 		winTemp + "worker.ign"
-	stdout, stderr, err := vm.Run(ignitionFileDownloadCmd, true)
+	out, err := vm.RunOverSSH(ignitionFileDownloadCmd, true)
 	if err != nil {
 		return errors.Wrap(err, "unable to download worker.ign")
 	}
 
-	log.V(5).Info("stderr associated with the ignition file download", "stderr", stderr)
-	if len(stderr) > 0 {
-		log.Info("error while downloading the ignition file from cluster", "stderr", stderr)
-	}
-	log.V(5).Info("stdout associated with the ignition file download", "stdout", stdout)
+	log.V(1).Info("ignition file download", "cmd", ignitionFileDownloadCmd, "output", out)
```

Comment:
Should this also be printed on error like done above for the WMCB?

Reply:
As in move the log before the error check?

Reply:
Why V(1) instead of V(5)?

Reply:
Same answer as before.